src/components/Article.jsx

```diff
@@ -62,17 +64,37 @@ if(loading){
             <p id="createdAt"> Created at: {getArticle.created_at}</p>
             </div>
             <h3 id="commentHeader">Comments</h3>
-            <section id="comments">
-                <div id="addCommentsWrapper">
-                    <div id="authorContainer">
-                <img id="authorAvatar" src={props.getUser.avatar_url}></img>
-                 <p id="author"> Username: {props.getUser.username}</p>
+            <section id="addComments">
+                    <div id="UserContainer">
+                        {
+                            props.getUser.username ? <img id="userAvatar" src={props.getUser.avatar_url}></img> : null
+                        }
+                        {
+                            props.getUser.username ?  <p id="username2"> Username: {props.getUser.username}</p> : <p id="noUsername"> Please choose a user to be able to make a comment</p>
+                        }
+               
                     </div>
-                </div>
-                <li id="comments list">
-
-                </li>
+                <label id="commentLable" htmlFor="commentArea">Comment Below</label>
+                <textarea id="commentArea"></textarea>
+               <button className="commentSubmitButton">Submit Comment</button>
             </section>
+                <div id="comments">
+                       { 
+                            commentsList.map(comment => {
+                                return <ul className="comment">
+                                    <label className="commentLables" htmlFor="commentId">Comment ID</label>
```

Comment:
When you're `map`ping over things in react, it's easy to forget something... perhaps an error in the browser console with tell us 👀

Reply:
added a key